### PR TITLE
run the project!

### PR DESCRIPTION
**Pull request by eebrukaya**
wrote Hello World!

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+<parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.0.1</version>
+    <relativePath/> <!-- lookup parent from repository -->
+</parent>
+
+<properties>
+    <java.version>17</java.version>
```
Comment (caglarcercinli) [inline pom.xml:19]: we need a jdk link I guess.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+</properties>
+<dependencies>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-security</artifactId>
```
Comment (caglarcercinli) [inline pom.xml:24]: why adding dependency before implementing?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    <dependency>
+        <groupId>org.springframework.security</groupId>
+        <artifactId>spring-security-test</artifactId>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
+
+<build>
+    <plugins>
+        <plugin>
```
Comment (caglarcercinli) [inline pom.xml:56]: i always wondered the role of plugin.

**Comment by caglarcercinli** [comment]
in the node repo we are using lint/prettier to analyse/format code. is there any equivelant for java projects?

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```
Comment (caglarcercinli) [inline pom.xml:5]: test comment

File: pom.xml (hunk 1 above)
Comment (eebrukaya) [reply pom.xml:19]: Ok, I'll add to readme.file

File: pom.xml (hunk 2 above)
Comment (eebrukaya) [reply pom.xml:24]: I need to use the dependency already in the future

File: pom.xml (hunk 3 above)
Comment (eebrukaya) [reply pom.xml:56]: because maven is not a library it should be added as a plugin. Maven is a package manager.

**Pull request merged**